src/color.rs

```diff
@@ -27,8 +27,25 @@ pub enum Color {
 }
 
 fn truecolor_support() -> bool {
-    let truecolor = env::var("COLORTERM");
-    truecolor.is_ok_and(|truecolor| truecolor == "truecolor" || truecolor == "24bit")
+    #[cfg(target_os = "windows")]
+    {
+        use windows_version::OsVersion;
+
+        // TrueColor support landed with Windows 10 build 14931,
+        // see https://devblogs.microsoft.com/commandline/24-bit-color-in-the-windows-console/
+        if OsVersion::current() >= OsVersion::new(10, 0, 0, 14931) {
+            true
+        } else {
+            false
+        }
```

Comment:
`if x then return true else return false` can be simplified to just `return x`.
```suggestion
        OsVersion::current() >= OsVersion::new(10, 0, 0, 14931)
```